utils/src/ooinstall/install_transactions.py

```diff
@@ -126,8 +126,20 @@ def run_main_playbook(hosts, hosts_to_run_on):
         facts_env['ANSIBLE_CONFIG'] = CFG.settings['ansible_config']
     return run_ansible(main_playbook_path, inventory_file, facts_env)
 
+
 def run_ansible(playbook, inventory, env_vars):
     return subprocess.call(['ansible-playbook',
                              '--inventory-file={}'.format(inventory),
                              playbook],
                              env=env_vars)
+
+def run_uninstall_playbook():
```

Comment:
seems a bit odd to have a run_uninstall_playbook in a install_transactions module.

Should this potentially be broken down further with a common module that contains run_ansible and other common methods and have install, uninstall and eventually upgrade modules that make use of the common module?